tests/pass/extern-so/call_extern_c_fcts.rs

```diff
@@ -0,0 +1,48 @@
+//@only-target-linux
+//@only-on-host
```

Comment:
I am curious, what happens when you run this on a non-host target (on a Linux host)?